pkg/module/go_get_fetcher_test.go

```diff
@@ -46,7 +56,7 @@ func ExampleFetch() {
 	repoURI := "github.com/arschles/assert"
 	version := "v1.0.0"
 	goBinaryName := env.GoBinPath()
-	fetcher := NewGoGetFetcher(goBinaryName, afero.NewOsFs())
+	fetcher, err := NewGoGetFetcher(goBinaryName, afero.NewOsFs())
```

Comment:
make sure you handle the error on this line, or just use `_` to show that it's intentional we don't want to handle the error :)